src/pages/help-centre/index.js

```diff
@@ -2,7 +2,7 @@ import React, { Component } from 'react'
 import { matchSorter } from 'match-sorter'
 import styled, { css } from 'styled-components'
 import { Helmet } from 'react-helmet'
-import { navigate } from '@reach/router'
+import { navigate } from 'gatsby'
```

Comment:
does this navigation localized?

Reply:
It's only used for navigating within the help-centre using anchor links.

Reply:
The function that utilises `navigate` here is not actually being called by the component. We can omit it in the future.

Reply:
The function that references `navigate` doesn't seem like is used anywhere. Should i remove this?

<img width="811" alt="Screenshot 2021-03-19 at 11 31 54 AM" src="https://user-images.githubusercontent.com/75345074/111727905-bf474600-88a6-11eb-91b0-6439d25393ae.png">

Reply:
we can remove that in the future